tool/tsh/common/tsh.go

```diff
@@ -687,6 +716,7 @@ const (
 	proxyKubeConfigEnvVar    = "TELEPORT_KUBECONFIG"
 	noResumeEnvVar           = "TELEPORT_NO_RESUME"
 	requestModeEnvVar        = "TELEPORT_REQUEST_MODE"
+	parallelJobsEnvVar       = "TELEPORT_PARALLEL_JOBS"
```

Comment:
Nit: "jobs" sounds like an overloaded term, I don't think we refer to these as jobs anywhere. Maybe `TELEPORT_PARALLEL_EXECS` or something?